lib/pipeline-notifications/chime.ts

```diff
@@ -0,0 +1,22 @@
+import { ChimeNotifier, ChimeNotifierOptions, IPipelineNotification, PipelineNotificationBindOptions } from '../';
+
+/**
+ * Properties to initialize ChimeNotification
+ */
+export interface ChimeNotificationProps extends ChimeNotifierOptions {
+}
+
+/**
+ * Notify events on pipeline to a Chime room.
+ */
+export class ChimeNotification implements IPipelineNotification {
+  constructor(private readonly props: ChimeNotificationProps) {
+  }
+
+  public bind(options: PipelineNotificationBindOptions): void {
+    new ChimeNotifier(options.pipeline, 'PipelineNotificationChime', {
```

Comment:
This way we can only have 1. Is that intentional?

Reply:
Not intended. Is that because of the SingletonLambda?